keps/prod-readiness/sig-api-machinery/3285.yaml

```diff
@@ -0,0 +1,6 @@
+# The KEP must have an approver from the
+# "prod-readiness-approvers" group 
+# of http://git.k8s.io/enhancements/OWNERS_ALIASES
+kep-number: 3285
+alpha:
+  approver: "@wojtek-t"
```

Comment:
You'll need to change to me if you want to get it in 1.25

Reply:
(if not you can leave it as Wojtek but please pull it from the enhacements sheet / mark as "Removed From Milestone").

Reply:
Changed the PRR approver to @johnbelamaric Thank you for being approver.